### PR TITLE
added default command flag

### PR DESCRIPTION
**Pull request by jdx**
Fixes https://github.com/heroku/cli-engine/issues/36

### CODE REVIEW

**Comment by KarateCowboy** [comment]
I'm going to add a test for this

**Comment by KarateCowboy** [comment]
[Default command as alias](https://trello.com/c/LUdXlGqg/1-default-command-as-alias)

**Comment by KarateCowboy** [comment]
[default command](https://trello.com/c/hQ57Pp37/2-default-command)

File: test/roots/test-foo/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "test-cli",
```
Comment (RasPhilCo) [inline test/roots/test-foo/package.json]: should we rename this directory to `tests/roots/test-cli`?

**Comment by RasPhilCo** [comment]
Just to NOTE: you can set more than one command to `default: true`, in which `Plugin` will just use [the first one it finds](https://github.com/heroku/cli-engine/blob/master/src/plugins.js#L164).

**Review by ransombriggs — APPROVED** [review]

**Pull request merged**